core/arch/arm/plat-imx/psci.c

```diff
@@ -61,7 +61,7 @@ int psci_cpu_on(uint32_t core_idx, uint32_t entry,
 	ns_entry_addrs[core_idx] = entry;
 
 	if (soc_is_imx7ds()) {
-		write32((uint32_t)CFG_TEE_LOAD_ADDR,
+		write32((uint32_t)TEE_TEXT_VA_START,
```

Comment:
This should write a physical address.
Either a virt2phys, or a comment stating this expects flat mapping.

Reply:
ok